packages/wdio-cli/src/utils.js

```diff
@@ -288,13 +288,26 @@ export async function generateTestFiles (answers) {
         process.cwd(),
         path.dirname(answers.specs).replace(/\*\*$/, '')
     )
+
     const destStepRootPath = path.join(process.cwd(), path.dirname(answers.stepDefinitions || ''))
 
+    let destPageObjectRootPath = ''
+
     const testFiles = answers.framework === 'cucumber'
         ? [path.join(TEMPLATE_ROOT_DIR, 'cucumber')]
         : [path.join(TEMPLATE_ROOT_DIR, 'mochaJasmine')]
 
     if (answers.usePageObjects) {
+        destPageObjectRootPath = path.join(
+            process.cwd(),
+            path.dirname(answers.pages).replace(/\*\*$/, '')
+        )
+        answers._relativePath =
+            (answers.generateTestFiles && answers.usePageObjects)
+                ? !(answers.framework === 'cucumber')
+                    ? path.relative(destSpecRootPath, destPageObjectRootPath)
+                    : path.relative(destStepRootPath, destPageObjectRootPath)
+                :  ''
```

Comment:
Let's move this into the `config.js` to have only 1 central place where we define which variables are available in the templates.

Also no need to start with an underscore for `_relativePath`. I had to do it for `async`/`await` because there was no other way.

Reply:
Done.